client/views/room/contextualBar/UserInfo/UserWebRTC/UserWebRTCWithData.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@rocket.chat/css-in-js';
+import { Box } from '@rocket.chat/fuselage';
+import colors from '@rocket.chat/fuselage-tokens/colors.json';
+import React, { ReactElement } from 'react';
+import { createPortal } from 'react-dom';
+
+import { IRoom } from '../../../../../../definition/IRoom';
+import { IUser } from '../../../../../../definition/IUser';
+import VerticalBar from '../../../../../components/VerticalBar';
+import { useWebRTC } from '../../../hooks/useWebRTC';
+import UserWebRTC from './UserWebRTC';
+import UserWebRTCButtons from './UserWebRTCButtons';
+
+const createAnchor = (): HTMLElement => {
```

Comment:
@ggazzo We have a similar function to create anchor for tooltip, I'm planning to create a helper function to receive the id of the `div` as parameter